src/EditorFeatures/VisualBasicTest/QualifyMemberAccess/QualifyMemberAccessTests.vb

```diff
@@ -220,6 +255,28 @@ End Class
 CodeStyleOptions.QualifyPropertyAccess)
         End Function
 
+        <WorkItem(28509, "https://github.com/dotnet/roslyn/issues/28509")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsQualifyMemberAccess)>
+        Public Async Function QualifyPropertyAccess_InObjectInitializer() As Task
```

Comment:
❕ This is a collection initializer, not an object initializer (test is correct, but the method name here is wrong)